### ISSUE TITLE
Appending reports

### ISSUE THREAD

**Issue by MightyPixel** [adamgruber/mochawesome#90]
Hi!

Is there a way to append the reports for multiple `mocha --reporter mochawesome` runs? Probably with different `--reporter-options`.

Cheers

**Comment by adamgruber** [comment]
Not currently. How would you envision this working? What's the final output you're looking for?

**Comment by MightyPixel** [comment]
In terms of the UI - see all suites that were run since the file was created. I'm not sure about the summary - if it isn't too much hassle aggregating them?

In terms of functionality - there could be a temporary file that is used to store the data between runs (the html itself could be used?). Or there could be a some tool that merges the results of multiple `mocha --reporter mochawesome` runs.

**Comment by adamgruber** [comment]
@MightyPixel The report was designed to present one test run only. Merging the data from multiple runs would produce unexpected results. Perhaps a better approach would be to add some kind of dashboard functionality. With v2 you can create a report from a JSON file separately from running the tests. That could be expanded to accept multiple JSON files and then create a sort of HTML dashboard page that would list out and link to the separate HTML reports that get generated. Would that be something of interest to you?

**Comment by MightyPixel** [comment]
Yes - this sounds good. The dashboard could be useful for other things as well like history data.

**Comment by mattflix** [comment]
Please see my related request in issue #172.

**Comment by adamgruber** [comment]
Same as #172 — merging/appending multiple runs is handled by [`mochawesome-merge`](https://www.npmjs.com/package/mochawesome-merge): it combines the per-run JSON files into one, which `marge` then renders as a single report. Closing as resolved by that tool.

---
_Triaged and posted via Claude Code, on behalf of @adamgruber._
